src/__tests__/paginate.query.builder.spec.ts

```diff
@@ -87,4 +88,28 @@ describe('Paginate with queryBuilder', () => {
     expect(result.meta.totalItems).toBe(9);
     expect(result.meta.totalPages).toBe(1);
   });
+
+  it('Can paginate with joins', async () => {
+    await connection
+      .createQueryBuilder()
+      .insert()
+      .into(TestRelatedEntity)
+      .values([
+        { id: 1, testId: 1 },
+        { id: 2, testId: 1 },
+        { id: 3, testId: 1 },
+      ])
+      .execute();
+
+    const qb = connection
+      .createQueryBuilder(TestEntity, 't')
+      .leftJoin('t.related', 'r');
+
+    const result = await paginate(qb, { limit: 5, page: 1 });
+
+    await connection.manager.delete(TestRelatedEntity, {});
```

Comment:
What's the delete query for?

Reply:
to clean the test data after the test I suppose